packages/contracts-bedrock/scripts/forges-test-names.ts

```diff
@@ -0,0 +1,77 @@
+import fs from 'fs'
```

Comment:
Should this script run as part of CI?

Reply:
eventually yes, but I haven't yet gone through all the test files.

Reply:
It would fail if run in CI now.